server/sso/index.js

```diff
@@ -3,124 +3,137 @@ const base64url = require('base64url');
 const cookie = require('cookie');
 const proxy = require('request');
 
-const config = {
+const CONFIG = {
   ssoAuth: {
-    host: 'www.ovh.com',
-    baseUrl: 'https://www.ovh.com/cgi-bin/crosslogin.cgi',
-    devLoginUrl: 'https://www.ovh.com/auth/requestDevLogin/',
+    eu: {
+      host: 'www.ovh.com',
+      baseUrl: 'https://www.ovh.com/cgi-bin/crosslogin.cgi',
+      devLoginUrl: 'https://www.ovh.com/auth/requestDevLogin/',
+    },
+    ca: {
+      host: 'ca.ovh.com',
+      devLoginUrl: 'https://ca.ovh.com/auth/requestDevLogin/',
+      baseUrl: 'https://ca.ovh.com/cgi-bin/crosslogin.cgi',
+    },
+    us: {
+      host: 'ovhcloud.com',
```

Comment:
No `us.` ? Could be nice to have some comments to understand why then 😄

Reply:
good catch